app/src/main/java/org/elasticsearch/bootstrap/BootstrapProxy.java

```diff
@@ -221,10 +203,14 @@ private void start() throws NodeValidationException {
         keepAliveThread.start();
     }
 
-    public static void stop() throws IOException {
+    public static void stopInstance() throws IOException {
+        INSTANCE.stop();
+    }
+
+    public void stop() throws IOException {
         try {
-            IOUtils.close(INSTANCE.node);
-            if (INSTANCE.node != null && INSTANCE.node.awaitClose(10, TimeUnit.SECONDS) == false) {
+            IOUtils.close(node);
```

Comment:
Minor: we removed INSTANCE as we made stop non-static and it's redundant - can we also remove `INSTANCE` from `INSTANCE.keepAliveLatch.countDown()`?